src/components/RadioButton/RadioButton.tsx

```diff
@@ -51,13 +52,21 @@ export interface RadioButtonState {
 }
 
 /** Radio button icon */
-const radioBtnIcon = (isFocused: boolean) => (
-  <div className={st(classes.radioIcon, isFocused ? classes.focused : '')}>
+const radioBtnIcon = (isFocused: boolean, isChildren: boolean) => (
+  <div
+    className={classnames(
+      classes.radioIcon,
+      {
+        [classes.withChildren]: isChildren,
```

Comment:
Note that you're using `classes.withChildren` on both the root and the icon element.
I would pass it only to the root element, that way you can target the icon element with it as well

Reply:
removed